arkouda-docker/README.md

```diff
@@ -8,10 +8,10 @@ The arkouda-full-stack image starts a one-locale arkouda\_server and provides an
 
 ```
 # set env variables
-export CHAPEL_SMP_IMAGE=chapel/chapel-gasnet-smp:1.28.0
-export ARKOUDA_BRANCH_NAME=2022.11.17
-export ARKOUDA_DISTRO_NAME=v2022.11.17
-export ARKOUDA_DOWNLOAD_URL=https://github.com/Bears-R-Us/arkouda/archive/refs/tags/v2022.11.17.zip
+export CHAPEL_SMP_IMAGE=chapel/chapel-gasnet-smp:1.29.0
+export ARKOUDA_BRANCH_NAME=2023.01.11
+export ARKOUDA_DISTRO_NAME=v2023.01.11
+export ARKOUDA_DOWNLOAD_URL=https://github.com/Bears-R-Us/arkouda/archive/refs/tags/v2023.01.11.zip
```

Comment:
is there a reason we're not updating to the `v2023.02.08` tag? Is it that we don't have a container for that version yet?

Reply:
@pierce314159 will do that tag next week 💯